op-node/metrics/metrics.go

```diff
@@ -452,9 +447,15 @@ func NewMetrics(procName string) *Metrics {
 
 // SetPeerScores updates the peer score [prometheus.GaugeVec].
 // This takes a map of labels to scores.
```

Comment:
Outdated doc comment.
Also worth adding that order of input slice doesn't matter

Reply:
Updated comment.  I also renamed the metric to just `peer_scores` instead of `peer_scores_histogram` - the `_histogram` was only appended initially to not conflict with the existing metric but now that it's gone we can go with the simpler name.  The histogram already appends things like `_bucket` etc to the individual metrics involved.